src/ScottPlot4/ScottPlot/SnapLogic/ISnap.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace ScottPlot.SnapLogic;
+
+public interface ISnap
```

Comment:
Is this interface used?

Reply:
`ScottPlot.SnapLogic.Independent2D` has contains `ISnap` objects